src/mahesh/FirstTest.java

```diff
@@ -0,0 +1,48 @@
+package mahesh;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class FirstTest {
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", ".//chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get("file:///D:/Technocredit/Project/TechnoGitProject/SeleniumBasics/SeleniumAssignment_1.html");
+		if (driver.getTitle().equals("TECHNOCREDITS")) {
+			WebElement firstName = driver.findElement(By.id("first name"));
+			firstName.sendKeys("Mahesh");
+			driver.findElement(By.id("last name")).sendKeys("Kumavat");
+			driver.findElement(By.id("E-mail")).sendKeys("MaheshKumavat@Gmail.com");
+			driver.findElement(By.id("Company Name")).sendKeys("Citi");
+			WebElement genderMale = driver.findElement(By.id("male"));
+			if(genderMale.isSelected())
```

Comment:
Here you did wrong validation, for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.